### PR TITLE
@86eqemm1q#WC- Contact us Change Text Field to Phone Field

### PR DESCRIPTION
**Pull request by emmanlemon**

### CODE REVIEW

File: apps/web-customer/src/components/blocks/ContactBlock/validation.ts [hunk 1]
```diff
@@ -8,6 +8,10 @@ export const contactSchema = yup.object({
     .matches(/^\d*$/, "Phone number must be a valid number")
     .required("Phone number is required")
     .default(""),
+  countryCode: yup
```
Comment (JMSevilla) [inline apps/web-customer/src/components/blocks/ContactBlock/validation.ts:11]: Adding this schema will look like it should be required by the user or it should automatically be occupied by our application. you can instead do this

phoneCode: yup.string().default(DEFAULT_PHONE_COUNTRY_CODE)

this `DEFAULT_PHONE_COUNTRY_CODE` can be found in `constant.ts`

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx [hunk 2]
```diff
@@ -27,11 +27,22 @@ export function ContactFormBlock() {
...
+    name: "countryCode",
+  });
+
   return (
     <ContactForm
       control={control}
       handleSubmit={handleSubmit}
       onSubmit={onSubmit}
+      handleSetCountryCode={handleSetCountryCode}
+      countryCode={countryCode}
```
Comment (JMSevilla) [inline apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx:45]: how about `form.watch('phoneCode')` much simpler and no need to initialize useWatch ?

**Review by JMSevilla — CHANGES_REQUESTED** [review]

File: apps/web-customer/src/components/blocks/ContactBlock/validation.ts (hunk 1 above)
Comment (emmanlemon) [reply apps/web-customer/src/components/blocks/ContactBlock/validation.ts:12]: The validation.ts file already has DEFAULT_PHONE_COUNTRY_CODE for country code.

File: apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx (hunk 2 above)
Comment (emmanlemon) [reply apps/web-customer/src/components/blocks/ContactBlock/ContactFormBlock.tsx]: I have already included a watch in the country code field.

**Review by acostales08 — APPROVED** [review]

**Review by JMSevilla — APPROVED** [review]

**Review by BryanPalad — APPROVED** [review]

File: apps/web-customer/src/components/blocks/ContactBlock/ContactForm.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
```
[the remark this answers is not in the document]
Comment (emmanlemon) [reply apps/web-customer/src/components/blocks/ContactBlock/ContactForm.tsx:6]: Added Comment on every file.

**Review by JMSevilla — APPROVED** [review]

**Pull request merged**